eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+import tseslint from 'typescript-eslint';
+import eslintConfigPrettier from "eslint-config-prettier";
+// import eslintConfigImport from 'eslint-plugin-import';
+
+export default tseslint.config(
+  {
```

Comment:
I'm not sure how `tseslint.config` works, so i might be overlooking something, but I think the `extends` section of the old config is missing/not migrated.

https://eslint.org/docs/latest/use/configure/migration-guide#predefined-and-shareable-configs